admin/sql/updates/20160310-mbs-4501-alternative-tracklists.sql

```diff
@@ -0,0 +1,134 @@
+\set ON_ERROR_STOP 1
+BEGIN;
+
+CREATE TABLE alternative_release (
+    id                      SERIAL,
+    gid                     UUID NOT NULL,
+    release                 INTEGER NOT NULL,
+    name                    VARCHAR,
+    artist_credit           INTEGER,
+    type                    INTEGER NOT NULL,
+    language                INTEGER NOT NULL,
+    script                  INTEGER NOT NULL,
+    comment                 VARCHAR(255) NOT NULL DEFAULT ''
+    CHECK (name != '')
+);
+
+CREATE TABLE alternative_release_type (
+    id                  SERIAL,
+    name                TEXT NOT NULL,
+    parent              INTEGER,
+    child_order         INTEGER NOT NULL DEFAULT 0,
+    description         TEXT,
+    gid                 UUID NOT NULL
+);
+
+CREATE TABLE alternative_medium (
+    id                      SERIAL,
+    medium                  INTEGER NOT NULL,
+    alternative_release     INTEGER NOT NULL,
+    name                    VARCHAR
+    CHECK (name != '')
+);
+
+CREATE TABLE alternative_track (
+    id                      SERIAL,
+    name                    VARCHAR,
+    artist_credit           INTEGER,
+    ref_count               INTEGER NOT NULL DEFAULT 0
+    CHECK (name != '' AND (name IS NOT NULL OR artist_credit IS NOT NULL))
```

Comment:
Yeah, exactly. The intent is that `NULL` means it should fallback to the original track's name or artist. But it doesn't make sense for both to be `NULL`.